src/components/common/DynamicTable.tsx

```
@@ -0,0 +1,179 @@
+                    String(column).toUpperCase()
+                  )}
+                </TableCell>
+              ))}
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {displayedData.map(row => (
+              <TableRow hover>
+                {columns.map(column => (
+                  <TableCell key={String(column)}>
+                    {String(row[column])}
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)